libs/ui-lib/lib/cim/components/ClusterDeployment/use-networking-formik.ts

```
@@ -47,32 +48,33 @@ const getInfraEnvProxy = (infraEnvs: InfraEnvK8sResource[]) => {
   initialValues: ClusterDeploymentNetworkingValues,
   hostSubnets: HostSubnets,
+  t: TFunction,
   openshiftVersion?: string,
 ) =>
   Yup.lazy((values: ClusterDeploymentNetworkingValues) =>
     Yup.object<ClusterDeploymentNetworkingValues>().shape({
-      clusterNetworks: clusterNetworksValidationSchema.when('stackType', {
+      clusterNetworks: clusterNetworksValidationSchema(t).when('stackType', {
         is: (stackType: NetworkConfigurationValues['stackType']) => stackType === IPV4_STACK,
-        then: () => clusterNetworksValidationSchema.concat(IPv4ValidationSchema),
+        then: () => clusterNetworksValidationSchema(t).concat(IPv4ValidationSchema),
```

Biome (2.3.13)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)